R/tableofdifferences.R

```diff
@@ -99,13 +99,25 @@
 #'     row labels specifying rows to remove from the returned table.
 #' @param column.names.to.remove Character vector or delimited string of
 #'     column labels specifying columns to remove from the returned table.
+#' @param means.test The type of test used to compare a table of means 
+#'  (i.e. from numeric variable). Usually taken from QSettings. 
+#'  One of "tTest", "zTest", "Nonparametric".  
```

Comment:
Can you move the links from the comment above `tTest` to the exported documentation for this function? e.g.
` #' @seealso \url{https://wiki.q-researchsoftware.com/wiki/Second_Order_Rao-Scott_Test_of_Independence_of_a_Contingency_Table}`

Reply:
I've moved the links to exported documentation in 72ff76b. I moved them into the description for the `means.test` and `proportions.test` parameters instead of a seealso link because that seemed like the more relevant place to look for it.